packages/debug/doguLogLevelRegistry.go

```diff
@@ -1,43 +1,49 @@
 package debug
 
 import (
+	"context"
 	"errors"
 	"fmt"
+	cesregistry "github.com/cloudogu/cesapp-lib/registry"
 	"gopkg.in/yaml.v3"
 )
 
 const keyDoguConfigLogLevel = "logging/root"
 
 type doguLogLevelYamlRegistryMap struct {
-	registry map[string]string
+	cesRegistry         cesregistry.Registry
+	doguReg             doguRegistry
+	logLevelRegistryMap map[string]string
 }
 
 // NewDoguLogLevelRegistryMap creates an instance of doguLogLevelYamlRegistryMap.
-func NewDoguLogLevelRegistryMap() *doguLogLevelYamlRegistryMap {
+func NewDoguLogLevelRegistryMap(cesRegistry cesregistry.Registry, doguReg doguRegistry) *doguLogLevelYamlRegistryMap {
 	return &doguLogLevelYamlRegistryMap{
-		registry: map[string]string{},
+		cesRegistry:         cesRegistry,
+		doguReg:             doguReg,
+		logLevelRegistryMap: map[string]string{},
 	}
 }
 
 // MarshalFromCesRegistryToString marshals the registry to yaml string.
-func (d *doguLogLevelYamlRegistryMap) MarshalFromCesRegistryToString(cesRegistry cesRegistry) (string, error) {
-	d.registry = map[string]string{}
+func (d *doguLogLevelYamlRegistryMap) MarshalFromCesRegistryToString(ctx context.Context) (string, error) {
+	d.logLevelRegistryMap = map[string]string{}
```

Comment:
Isn't this already initialized by the constructor?

Reply:
that's correct ^^'